src/controllers/additional_info.js

```diff
@@ -0,0 +1,47 @@
+const { saveSection, getSection } = require('../model/form_queries');
+
+exports.get = (req, res) => {
+  getSection(req.session.id, 'additional')
+    .then((data) => {
+      res.render('additional_info', {
+        activePage: { additional_info: true },
+        errorMessages: req.flash('error'),
+        successMessages: req.flash('success'),
+        pageTitle: 'Additional Information',
+        logoutButton: true,
+        progressPercentage: '80',
+        data,
+      });
+    })
+    .catch((err) => {
+      console.log(err);
+      req.flash('error', 'Sorry - we couldn\'t load your saved answers for this section');
+      res.render('additional_info', {
+        activePage: { additional_info: true },
+        errorMessages: req.flash('error'),
+        successMessages: req.flash('success'),
+        pageTitle: 'Additional Information',
+        progressPercentage: '80%',
```

Comment:
The '%' is showing up twice in our progress bar - maybe remove it from here as I think it must be in the progress bar partial. Also the percentage values will need changing on every form page as the form is now longer

Reply:
:+1: